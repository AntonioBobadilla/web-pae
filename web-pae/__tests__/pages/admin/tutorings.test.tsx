import Tutorings from '@/pages/admin/tutorings';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Tutorings', () => {
  it('renders tutorings page unchanged', () => {
    const { container } = render(<Tutorings />);
    expect(container).toMatchSnapshot();
  });
});
