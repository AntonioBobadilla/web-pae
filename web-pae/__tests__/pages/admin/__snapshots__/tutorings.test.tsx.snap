// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders tutorings page unchanged 1`] = `
<div>
  <h1>
    Agendar asesoria
  </h1>
</div>
`;

exports[`Tutorings renders tutorings page unchanged 1`] = `
<div>
  <h1>
    Agendar asesoria
  </h1>
</div>
`;
